tests/copilot_helper/utils/fixtures/addons_files/opensearch_addons_bad_data.yml

```diff
@@ -3,52 +3,48 @@ my-opensearch-1:
   nonsense: abc
 
 my-opensearch-2:
-  type: opensearch
-  deletion_policy: 27
-
-my-opensearch-3:
   type: opensearch
   environments: False
 
-my-opensearch-4:
+my-opensearch-3:
```

Comment:
Could these have meaningful suffixes and remove the need to adjust them all is we add or remove one?